flink-tests/src/test/java/org/apache/flink/test/streaming/runtime/BroadcastStateITCase.java

```diff
@@ -145,17 +147,16 @@ public Watermark checkAndGetNextWatermark(T lastElement, long extractedTimestamp
 		private static final long serialVersionUID = 7616910653561100842L;
 
 		private final Map<Long, String> expectedState;
-
-		private final long timerTimestamp;
+		private final AtomicLong al = new AtomicLong(1000L);
+		// <key, time>
+		private final Map<Long, Long> expectedTimeAndKey = new HashMap<>();
```

Comment:
I think the name `timerToExpectedKey` would better express the intention and the comment `// <key, time>` is redundant.